events-api/src/modules/tickets/tickets.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { validateRequestBody } from "#/shared/validators/request-body.validator";
+import { Router } from "express";
+import { ticketsCreateDtoSchema } from "./dto/requests/tickets-create-dto";
+import { TicketsService } from "./tickets.service";
+import { validateRouteParams } from "#/shared/validators/route-params.validator";
+import { eventsRouteParamsDtoSchema } from "../events/dto/requests/events-route-params.dto";
+
+export const TicketsController = Router();
+
+TicketsController.post(
+  "/",
+  validateRequestBody(ticketsCreateDtoSchema),
+  async (req, res) => {
+    await TicketsService.createTicket(req.body);
+
+    return res.status(201).json({
+      message: "Tickets created successfully",
+    });
+  }
+);
+
+TicketsController.get(
+  "/:id",
+  validateRouteParams(eventsRouteParamsDtoSchema),
+  async (req, res) => {
+    const ticket = await TicketsService.getTicket(req.params["id"] as string);
```

Comment:
What does happen if the ticket is not found?

Reply:
![image](https://github.com/user-attachments/assets/44b131f5-e404-4ed1-a146-f550414e249c)
handeled